sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Generated/AuthenticationRestClient.cs

```diff
@@ -156,10 +156,10 @@ internal HttpMessage CreateExchangeAcrRefreshTokenForAcrAccessTokenRequest(strin
         /// <param name="service"> Indicates the name of your Azure container registry. </param>
         /// <param name="scope"> Which is expected to be a valid scope, and can be specified more than once for multiple scope requests. You obtained this from the Www-Authenticate response header from the challenge. </param>
         /// <param name="refreshToken"> Must be a valid ACR refresh token. </param>
-        /// <param name="grantType"> Grant type is expected to be refresh_token. </param>
+        /// <param name="grantType"> Grant type is expected to be refresh_token. The default value is AutoRest.CSharp.Output.Models.Types.EnumTypeValue. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="service"/>, <paramref name="scope"/> or <paramref name="refreshToken"/> is null. </exception>
-        public async Task<Response<AcrAccessToken>> ExchangeAcrRefreshTokenForAcrAccessTokenAsync(string service, string scope, string refreshToken, TokenGrantType grantType = TokenGrantType.RefreshToken, CancellationToken cancellationToken = default)
+        public async Task<Response<AcrAccessToken>> ExchangeAcrRefreshTokenForAcrAccessTokenAsync(string service, string scope, string refreshToken, TokenGrantType grantType, CancellationToken cancellationToken = default)
```

Comment:
Why do we need to remove the default value for `grantType` here?

Reply:
It is client default value. We will remove it.

Reply:
But why? I still haven't heard a reason why this is being done. We set the default to ease the developer experience. This makes it harder.